internal/runbits/dependencies/changesummary.go

```diff
@@ -18,23 +18,27 @@ import (
 // dependency numbers.
 const showUpdatedPackages = true
 
-// OutputChangeSummary looks over the given artifact changeset and attempts to determine if a single
-// package install request was made. If so, it computes and lists the additional dependencies being
-// installed for that package.
-// `artifacts` is an ArtifactMap containing artifacts in the changeset, and `filter` contains any
-// runtime requirements/artifacts already installed.
-func OutputChangeSummary(out output.Outputer, changeset *buildplan.ArtifactChangeset, alreadyInstalled buildplan.Artifacts) {
+// OutputChangeSummary looks over the given build plans, and computes and lists the additional
+// dependencies being installed for the requested packages, if any.
+func OutputChangeSummary(out output.Outputer, newBuildPlan *buildplan.BuildPlan, oldBuildPlan *buildplan.BuildPlan) {
+	requested := newBuildPlan.RequestedArtifacts()
+	if len(requested) == 0 {
+		return // nothing to do
+	}
+
 	addedString := []string{}
 	addedLocale := []string{}
-	added := buildplan.Ingredients{}
+	for _, i := range requested.Ingredients() {
+		v := fmt.Sprintf("%s@%s", i.Name, i.Version)
+		addedString = append(addedLocale, v)
+		addedLocale = append(addedLocale, fmt.Sprintf("[ACTIONABLE]%s[/RESET]", v))
+	}
```

Comment:
This doesn't seem appropriate. This would give you all requested ingredients for a buildplan, not just the ones that were added.

Reply:
Ok but we shouldn't be pushing something that's not fixing the issue, especially when it introduces its own issues. This is creating another bug on top of the existing one.

Suggest dropping this and maybe adding some context to the description of DX-2851.

Reply:
https://activestatef.atlassian.net/browse/DX-2851?focusedCommentId=52777